src/spec-node/featuresCLI/package.ts

```diff
@@ -0,0 +1,98 @@
+import path from 'path';
```

Comment:
Consider converting newly added `.ts` files to use tabs instead of spaces to align with the current sources (`package.ts`, `packageCommandImpl.ts`, `test.ts`, `testCommandImpl.ts`, maybe some the file containing the actual test under `src/test` ?).

This doesn't seem to be enforced in the CI, but it might be good to try to keep things consistent until we add something to enforce it. (we could add `"editor.insertSpaces": false` to the `.vscode/settings.json` file, which will make newly created files default to using tabs in vscode).